name/feeds.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.syndication.views import Feed
+from django.core.urlresolvers import reverse_lazy
+from django.utils.feedgenerator import Atom1Feed
+
+from . import app_settings
+from .models import Name
+
+
+class NameAtomFeedType(Atom1Feed):
+    """Custom Atom Feed Type for Name objects.
+
+    This will set the Content-Type header to application/xml, and
+    add a georss:point element to any entry that has a current location
+    set.
+    """
+    mime_type = 'application/xml'
+    GEO_RSS_NS = 'http://www.georss.org/georss'
+
+    def root_attributes(self):
+        attrs = super(NameAtomFeedType, self).root_attributes()
+        attrs['xmlns:georss'] = self.GEO_RSS_NS
+        return attrs
+
+    def add_item_elements(self, handler, item):
+        super(NameAtomFeedType, self).add_item_elements(handler, item)
+
+        # Add the georss:point element if the item has a geo_point
+        # attribute.
+        geo_point = item.get('geo_point', None)
```

Comment:
I think we should change this to take the comma out of the geo_point http://www.georss.org/ @vphill @damonkelley